deploy/charts/istio-csr/templates/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cert-manager-istio-agent.name" . }}
+  labels:
+{{ include "cert-manager-istio-agent.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ include "cert-manager-istio-agent.name" . }}
```

Comment:
Can we add more labels to match on?

Reply:
What else would you like to match on?

Reply:
need to look at what we do in cert-manager (core) i see manuy re-use the labels helper